### ISSUE TITLE
A11y_Azure Data Studio_Nov2023_Database Properties Preview_Files_Keyboard: When navigating to Files "Database files" grid column headers is not accessible with keyboard.

### ISSUE THREAD

**Issue by HindujaArugula** [microsoft/azuredatastudio#25137]
"Please do not close this bug. This bug should only be closed by TCS, C+AI Accessibility or the Divisional Driver."
[Check out Accessibility Insights!](https://nam06.safelinks.protection.outlook.com/?url=https://accessibilityinsights.io/&data=02%7c01%7cv-stfe%40microsoft.com%7cb67b2c4b646d4f9561a208d6f4b5c39b%7c72f988bf86f141af91ab2d7cd011db47%7c1%7c0%7c636965458850501301&sdata=mxhokIKNMb22llsjXHLgU3XZibj1Qfx37rpY4PU2sfE%3D&reserved=0) - Identify accessibility bugs before check-in and make bug fixing faster and easier.

### Git hub Tags:
#A11yTCS; #A11yMAS; #A11ySev1; #DesktopApp; #WCAG2.1.1; #FTP; A11yTCS, #SH_Azure Data Studio_Nov2023_Mac_Dec2023; #SQL Azure Data Studio; #MAC; #Keyboard;

### Environment Details:
Application Name: Azure Data Studio
Operating System: MAC
Azure data studio 1.47.0

### Repro Steps:

1. Install Azure data studio application and open it
2. Make sure to enable "Preview" feature in settings.
3. Tab till "Database" and press ENTER to expand the tree.
4. Tab till any database and pressF10 to open context menu.
5. Tab till "Properties Preview" and press ENTER.
6. Tab till "Files" tab and press ENTER.
7. Tab till "Database files" table and try to change column size.
8. You will observe that when navigating to Files "Database files" grid column headers is not accessible with keyboard.

### Actual Result:
When navigating to Files "Database files" grid column headers is not accessible with keyboard.

**Note: Issue seen throughout the application for similar type control.**

### Expected Result:
When navigating to Files "Database files" grid column headers should be accessible with keyboard & also should be able to adjust the column header with keyboard.

### User Impact:
Keyboard only users will face difficulties if the focus does not go to column header and cannot resize the column header.

### Attachment:
![Keyboard grid (1)](https://github.com/microsoft/azuredatastudio/assets/125462007/a404e1c0-11b2-48f8-994c-5f98258b0a4e)

**Comment by corivera** [comment]
@HindujaArugula resizing columns is provided via the keyboard shortcut Shift+Alt+S. Is that sufficient to resolve this issue?

**Comment by HindujaArugula** [comment]
Yes, it fine but make sure to document it so the end user will know how to navigate using shortcut keys Shift+Alt+S.

**Comment by corivera** [comment]
@HindujaArugula Are we good to close this issue now? You can verify the behavior with this Mac build of ADS [here](https://azuredatastudio-update.azurewebsites.net/latest/darwin-universal/stable)

**Comment by HindujaArugula** [comment]
@corivera Tried to check the issue using shortcut keys which mentioned in the above bug, but we are not able take action using shortcut keys.
Could you please help us with this issue?

**Comment by kisantia** [comment]
@HindujaArugula does the keyboard shortcut of shift+option+s work for you on mac? A quickpick should appear where you can specify the column width, as seen below:
![resizeColumn](https://github.com/microsoft/azuredatastudio/assets/31145923/5dc9f3a3-aab7-42c4-b85e-ea5bccc028d9)

**Comment by HindujaArugula** [comment]
Verified the issue in the below mac build.
macos: sonoma 14.2.1
 Version: 1.48.0-insider (Universal)
Commit: 7bdf5c994bcc25aaeb60584dcaa24f83e6fde7fd
Date: 2024-01-29T07:04:50.019Z
VS Code: 1.82.0
Electron: 25.9.8
Chromium: 114.0.5735.289
Node.js: 18.15.0
V8: 11.4.183.29-electron.0
OS: Darwin arm64 23.2.0

Issue is fixed and does not repro. Hence closing the bug.

https://github.com/microsoft/azuredatastudio/assets/125462007/7b7ffce7-eb61-4883-8958-d5e9ef34e714
